image_width: 640
image_height: 480
camera_name: front_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [498.360614277498, 0, 317.3304820509922, 0, 499.1540096847331, 248.9065792381122, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.06053031484188461, -0.1577409368340061, 0.002175590574414737, -0.000729499814666171, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [497.2951354980469, 0, 316.6855409715336, 0, 0, 501.8094177246094, 249.7164076354038, 0, 0, 0, 1, 0]